packages/contracts-bedrock/interfaces/universal/ICrossDomainMessenger.sol

```diff
@@ -17,6 +17,9 @@ interface ICrossDomainMessenger {
     function RELAY_CONSTANT_OVERHEAD() external view returns (uint64);
     function RELAY_GAS_CHECK_BUFFER() external view returns (uint64);
     function RELAY_RESERVED_GAS() external view returns (uint64);
+    function TX_BASE_GAS() external view returns (uint64);
+    function FLOOR_CALLDATA_OVERHEAD() external view returns (uint64);
+    function ENCODING_OVERHEAD() external view returns (uint64);
```

Comment:
These constants should be internal with camel case getters exposed, unless we are intentionally doing this for compatibility with the other related constants?

Reply:
Yeah, doing it intentionally.